src/components/SettingsGrid/SettingsGridCard/index.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+import SettingsGridCard from './SettingsGridCard'
+
+export default SettingsGridCard
```

Comment:
better to use shortened version for filenames - just Card.js
but left SettingsGridCard declarations (convenient to reading and support)
please see another nested components for the reference

Reply:
I guess, it's not a better way, furthermore it's more confusing, when your component name is difference of filename.